scripts/run_performance_tests.py

```diff
@@ -0,0 +1,165 @@
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS-IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Runs the performance tests."""
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import argparse
+import atexit
+import os
+import subprocess
+import time
+
+import python_utils
+
+from . import clean
+from . import common
+from . import install_third_party_libs
+from . import setup
+from . import setup_gae
+
+_PARSER = argparse.ArgumentParser(
+    description='The root folder MUST be named \'oppia\'. '
```

Comment:
Yeah I really don't like these after all, this will just print a huge blob of text. Please use triple strings instead so we can have proper newlines/indentations/etc.

Reply:
Done